sys-config/ltoize/files/package.cflags/ltoworkarounds.conf

```diff
@@ -125,6 +125,10 @@ media-libs/flac *FLAGS+=-fno-ipa-cp-clone # -fipa-cp-clone is enabled by default
 media-libs/faad2 *FLAGS+=-fno-tree-loop-vectorize # causes subtly wrong decoding
 # END: Deliberate -O3 workarounds
 
+# BEGIN: -Ofast workarounds
+dev-lang/python C*FLAGS+='-fno-finite-math-only' # instrumentation tests hang/segfault during emerge
```

Comment:
Just a clarification -- is C*FLAGS necessary or will *FLAGS work too?

Reply:
oh *FLAGS will work too

Reply:
should i change it to that?

Reply:
If you don't mind -- it's just to keep things in line with the rest of the workarounds so that later it doesn't confuse anyone.

Reply:
sure no worries